utils/clients/tools/isisicat_prefix_mapping.py

```diff
@@ -0,0 +1,44 @@
+# ############################################################################### #
+# Autoreduction Repository : https://github.com/ISISScientificComputing/autoreduce
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI
+# SPDX - License - Identifier: GPL-3.0-or-later
+# ############################################################################### #
+"""
+fetch_instrument_fullname_mapping() can be used to map Autoreduction to ICAT instrument prefixes
+"""
+from utils.clients.icat_client import ICATClient
+from utils.settings import VALID_INSTRUMENTS as AUTOREDUCTION_INSTRUMENT_NAMES
+from utils.clients.tools.isisicat_prefix_mapping_logging_setup import logger
+
+
+def fetch_instrument_fullname_mapping():
```

Comment:
NITPICK: Optionally, this method could take an optional argument such as a list of instruments, to return prefixes only for those instruments instead of returning all instruments when called. 
As there are only around 30 instruments in total (and only 10 using AR right now - could grow to 30 eventually) or so for ISIS, whether or not you make this change is entirely up to you.
My thinking here is that you could call the method with not arguments to get all mappings or call it with a list of N instruments to return mappings for N instruments only. This would mean  you don't have to filter through the returned values as you should get exactly what you are expecting.